spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/Sampler.java

```diff
@@ -18,10 +18,12 @@
 
 /**
  * Extremely simple callback to determine the frequency that an action should be traced.
+ *
+ * @since 1.0.0
  */
 public interface Sampler {
 	/**
-	 * @param span the current span (or null if there is none)
+	 * @param span the current Span (or null if there is none)
```

Comment:
I don't think this doc belongs here, as the sampler doesn't know about what's current or not.

maybe `/** returns true if the span is not null and should be exported to the tracing system */`